src/main/kotlin/org/coepi/api/v4/dao/TCNReportRecord.kt

```diff
@@ -0,0 +1,21 @@
+package org.coepi.api.v4.dao
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable
+
+@DynamoDBTable(tableName = "TCNReports")
```

Comment:
If the `hash key` is still reportId and the `range key` is still `reportTimestamp` then all that's needed to update the Terraform provisioning is to change the table and column names to the new versions in `terraform/dynamodb.tf`

If possible I'd like to do that in this MR so that we commit all the related v0.4 changes together.

Reply:
range key for the new table is randomId, which is a randomly generated GUID, this is done to address https://github.com/Co-Epi/coepi-backend-aws/issues/1

The existing TF stack is still valid for v3. I need a way to create both stacks with as much re-usability as possible. We can remove the v3 stuff when we deprecate it in the near future.

Reply:
Updated TF template